Makefile.features

```diff
@@ -0,0 +1,8 @@
+# import list of provided features
+-include $(RIOTBOARD)/$(BOARD)/Makefile.features
+-include $(RIOTCPU)/$(CPU)/Makefile.features
+
+DEFAULT_FEATURES += periph_pm
+
+# add available default features to required list
+FEATURES_REQUIRED += $(filter-out $(DISABLE_FEATURES), $(filter $(FEATURES_PROVIDED), $(DEFAULT_FEATURES)))
```

Comment:
Wait, I think there is a conceptual error here: 
`FEATURES_REQUIRED != (FEATURES_PROVIDED - DISABLE_FEATURES)`
`FEATURES_REQUIRED` should filled by the dependency definitions in the `Makefile.dep`s, e.g. the `at86rf2xx` requires `periph_gpio` and `periph_spi`.

So I guess this should be something like `FEATURES_REAL` := `FEATURES_REQUIRED` + `DEFAULT_FEATURES` - `DISABLE_FEATURES`?!